server/build.gradle.kts

```diff
@@ -27,6 +27,9 @@ dependencies {
   implementation(libs.bundles.jersey)
   implementation(libs.metrics.jersey2)
 
+  // Workaround for the task copyActivation.
+  compileOnly(libs.javax.activation)
```

Comment:
Why it is compileOnly and add additional action to copy jar?

Reply:
No, in fact both runtime and compile stage did not depend on it, what I need is just to copy `activation.jar` to the package.  

Using `implementation` will make the CI fail in the embedd mode.